src/travel-details-screens/utils.ts

```diff
@@ -160,6 +161,7 @@ export function hasShortWaitTime(legs: Leg[]) {
 }
 
 export function hasShortWaitTimeAndNotGuaranteedCorrespondence(legs: Leg[]) {
+  if (hasGuaranteedCorrespondence(legs)) return false;
```

Comment:
If there is a non-corresponding short wait time between two legs of the trip, then this method should return `true`. I think this now may be masked by some other corresponding legs, which overrides the answer to `false`.